docs/included.md

```diff
@@ -13,6 +13,10 @@
   - [API](api.md#scores.continuous.additive_bias)
   - [Tutorial](project:./tutorials/Additive_and_multiplicative_bias.md)
   - [Mean Error (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#meanerror)
+* - Additive Bias percentage
```

Comment:
```suggestion
* - Additive Bias Percentage
```

Reply:
Capitalisation (for consistency with the other entries in included.md)

Reply:
After some more background reading, this score is clearly known universally as "Percent bias", so I suggest we list it as "Percent Bias (PBIAS)" (rather than additive percentage bias), despite my earlier thoughts on the matter. This will also then need to be relocated in the table to preserve alphabetical ordering.